packages/block-library/src/query/block.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "core/query",
+	"category": "layout",
+	"attributes": {
+		"id": {
+			"type": "number"
+		},
+		"query": {
+			"type": "object",
+			"default": {
+				"per_page": 3,
+				"pages": 1,
```

Comment:
What does "pages" refer to here? The max pages?

Reply:
Yes